src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1881,10 +1870,7 @@ void SolverInterfaceImpl::clearMappings(utils::ptr_vector<MappingContext> contex
   // Clear non-stationary, non-incremental mappings
```

Comment:
```suggestion
```

Reply:
I don't really understand how `SolverInterfaceImpl::clearMappings()` is related to this PR. Why should we remove it? If we remove the calls of `SolverInterfaceImpl::clearMappings()` from `mapWrittenData` and `mapReadData`, we can remove the entire function as far as I understand.

Reply:
Btw: This is something @fsimonis might also veto, because I remember him having some good experiences with resetting data of mappings at the right point in time.

Reply:
The `onadvance` option basically meant that the mapping got cleared after every call. As we no longer have this option, we neither need the call. We basically make the previous `initial` option our default.
`clearMappings`, we only need after the repartitioning.